### PR TITLE
Update key-mgnt diagram

### PR DESCRIPTION
**Pull request by mbrandenburger**
**What this PR does / why we need it**:
- Split Admin and Peer_CLI
- Rename component to sync with other diagrams
- Rename functions to keygenerate, keyexport, and keyimport to better match new mgnt api
- Added sealing using external builders magic super powers
- Add a note to clarify where MVP flow ends
- Simplify diagram by removing ORG2;
- Addresses open review comments from #375 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing changes and/or breaks backward compatability?**:

```

### CODE REVIEW

File: docs/design/fabric-v2+/fpc-key-dist.puml [hunk 1]
```diff
@@ -180,125 +179,147 @@ group chaincode key distribution
...
-    chaincode key registration (same as that from Admin1)
-  end ref
+  Enclave1      -> Enclave1         :   verify Sig_Enclave1
+  Enclave1      -> Enclave1         :   SEK, Chaincode_DK <- decrypt(cckeydist_start_message.encrypted_keys_message) with Enclave2_DK
+  Enclave1      -> Enclave1         :   extract Chaincode_EK from Chaincode_DK
+  Enclave1 -> Enclave1 : sealed_cckeys <- seal <SEK, Chaincode_EK, Chaincode_DK>
+  Enclave1      -> Enclave1         :   SIG_Enclave2 <- sign Chaincode_EK with Enclave2_SK
+  Enclave1      -> Enclave1         :   cckey_registration_message <- <Enclave2_VK, Sig_Enclave2, Chaincode_EK>
+
+  return cckey_registration_message /'Enclave2 -> ECC2'/
```
Comment (g2flyer) [inline docs/design/fabric-v2+/fpc-key-dist.puml:306]: You have to return sealed_cckeys and store it here.
BTW: given that we have now two separate sealed blobs, would be nice to mention both explicit and with same name as here and in regtration  in fpc-cc-invocation?

File: docs/design/fabric-v2+/fpc-key-dist.puml [hunk 2]
```diff
@@ -49,125 +39,134 @@ note over Admin1, Admin2
...
-    generation using the local enclave.
-    This might as well be Admin2
-    using Enclave2.
+    One admin must begin the key generation using the local enclave.
+    This might as well be Admin2 using Enclave2.
   end note
-  Admin1        -> Admin1       :   fill CC_Parameters structure
-  Admin1        -> Peer1_CLI    ++: GenerateCCKeys(CC_Parameters)
+  Admin1        -> Peer1_CLI    ++: generateCCKeys(CC_NAME)
+
```
[the remark this answers is not in the document]
Comment (mbrandenburger) [reply docs/design/fabric-v2+/fpc-key-dist.puml:47]: You are right, let me fix that

File: docs/design/fabric-v2+/fpc-key-dist.puml [hunk 3]
```diff
@@ -49,125 +39,134 @@ note over Admin1, Admin2
...
-  return cckey_registration_message /'Admin1 -> Peer1_CLI'/
+  Enclave1 -> Enclave1 : sealed_cckeys <- seal <SEK, Chaincode_EK, Chaincode_DK>
 
-end group
+  return cckey_registration_message, selaed_cckeys /'ECC1 -> Enclave1    '/
+  ECC1 -> ECC1 : store sealed cc_keys
+  note right ECC1
+     FPC_stub stores sealed cckeys with the help of
+     the external builder on the peer local storage.
+  end note
```
[the remark this answers is not in the document]
Comment (mbrandenburger) [reply docs/design/fabric-v2+/fpc-key-dist.puml:87]: good catch! Removed FPC_manager from invocation diagram in my other open PR

File: docs/design/fabric-v2+/fpc-key-dist.puml [hunk 4]
```diff
@@ -49,125 +39,134 @@ note over Admin1, Admin2
...
         if false, abort
     end note
     ERCC1   -> ERCC1         :  extract CC_Id from Credentials
-    ERCC1   -> ERCC1         :  Registered_Chaincode_EK <- get_state(CC_Id."Chaincode_EK")
-    ERCC1   -> ERCC1         :  if(!Registered_Chaincode_EK) put_state(CC_Id."Chaincode_EK", cckey_registration_message.Chaincode_EK)
-    ERCC1   -> ERCC1         :  else check Registered_Chaincode_EK and cckey_registration_message.Chaincode_EK match
-    ERCC1   -> ERCC1         :  store cckey_registration_message on the ledger
-    note over ERCC1
-        E.g., put_state(key=<"keydist_registered".Enclave1_VK>, value=cckey_registration_message)
+    ERCC1   -> ERCC1         :  cc_ek <-
```
[the remark this answers is not in the document]
Comment (mbrandenburger) [reply docs/design/fabric-v2+/fpc-key-dist.puml:125]: Ahh removed!

File: docs/design/fabric-v2+/fpc-key-dist.puml (hunk 1 above)
Comment (mbrandenburger) [reply docs/design/fabric-v2+/fpc-key-dist.puml:306]: right! good catch!

**Comment by g2flyer** [comment]
Moved assignment to bruno as he is the missing aproval.  (That said, given that there are fixup, Marcus has to get anyway in the loop again after the approval for the autosquash, so i guess marcus could merge in that case as well directly ...)

File: docs/design/fabric-v2+/fpc-key-dist.puml [hunk 5]
```diff
@@ -180,125 +186,153 @@ group chaincode key distribution
...
+  note right Enclave1
     Regarding Enclave2's chaincode definition, we can only check that CC_Id matches,
     because the other fields might have changed.
     Alternatively, we could access the "current" chaincode definition,
     and check whether Enclave2's matches it.
   end note
   Enclave1      -> Enclave1         :   check Enclave2_Credentials.Attested_Data.CC_Parameters.CC_Id against internal one
-  Enclave1      -> Enclave1         :   check Enclave2's keys and Enclave2_VK, Enclave2_EK, attestation_report
-  Enclave1      -> Enclave1         :   encrypted_cckeys_message <- encrypt <SEK, Chaincode_DK> with Enclave2_EK
+  Enclave1      -> Enclave1         :   check Enclave2_VK, Enclave2_EK, ATTESTATION
```
Comment (bvavala) [inline docs/design/fabric-v2+/fpc-key-dist.puml:211]: (follow up on a comment of Michael's)
```suggestion
  note right Enclave1
    The attestation verification requires to have the IAS keys inside the enclave.
    Alternatively, this check can be removed, thus relying on ERCC and TLCC.
    ERCC already verified the attestation as part of the enclave registration,
    using the IAS keys which are part of ERCC itself.
    Also, TLCC previously confirmed that such registration was committed.
  end note
  Enclave1      -> Enclave1         :   check Enclave2_VK, Enclave2_EK, ATTESTATION
```

Comment (g2flyer) [reply docs/design/fabric-v2+/fpc-key-dist.puml:211]: didn't notice that attestation was still there. good point to add that

File: docs/design/fabric-v2+/fpc-key-dist.puml [hunk 6]
```diff
@@ -12,162 +12,168 @@
...
+group chaincode key export
+  note right Admin1
+    if necessary, upgrade chaincode policy to enable target Enclave endorsements
   end note
 
-  Admin1        -> ERCC1          ++: query registered enclave's key
-  return list of enclave Credentials /'Admin1 -> ERCC1'/
-  Admin1        -> Admin1           : select non-provisioned Enclave2's Credentials from list
-  note over Admin1
+  Admin1 -> Peer1_CLI ++: exportCCKeys(CC_Id)
```
Comment (g2flyer) [inline docs/design/fabric-v2+/fpc-key-dist.puml:158]: For consistency with createenclave
```suggestion
 Admin1 -> Peer1_CLI ++:  lifecycle chaincode\n exportcckeys CC_Id
```

File: docs/design/fabric-v2+/fpc-key-dist.puml [hunk 7]
```diff
@@ -180,125 +186,160 @@ group chaincode key distribution
...
-end group
+group chaincode key import
+  note right Admin1
+    NOTE that import is called at another peer to provision a FPC enclave with
+    the keys exported in the chaincode key export flow above.  In order to simplify
+    this diagram, only a single admin, peer, enclave, ercc is depicted even though
+    the import flow may involve other entities or the target / recipient enclave.
+  end note
+
+  Admin1        -> Peer1_CLI        ++: importCCKeys(CC_Id)
```
Comment (g2flyer) [inline docs/design/fabric-v2+/fpc-key-dist.puml:281]:
```suggestion
  Admin1        -> Peer1_CLI        ++: lifecycle chaincode\n importcckeys (CC_Id)
```

**Pull request merged**